src/erlcloud_sns.erl

```diff
@@ -286,8 +288,8 @@ list_endpoints_by_platform_application(PlatformApplicationArn, NextToken, Access
     list_endpoints_by_platform_application(PlatformApplicationArn, NextToken, new_config(AccessKeyID, SecretAccessKey)).
 
 -spec list_topics/0 :: () -> [{topics, [[{arn, string()}]]} | {next_token, string()}].
--spec list_topics/1 :: (undefined | string() | aws_config()) -> [{topics, [[{arn, string()}]]} | {next_token, string()}].
--spec list_topics/2 :: (undefined | string(), aws_config()) ->  [{topics, [[{arn, string()}]]} | {next_token, string()}].
+-spec list_topics/1 :: (undefined | parameter() | aws_config()) -> [{topics, [[{arn, string()}]]} | {next_token, string()}].
+-spec list_topics/2 :: (undefined | parameter(), aws_config()) ->  [{topics, [[{arn, string()}]]} | {next_token, string()}].
```

Comment:
Line 292 is too long: -spec list_topics/2 :: (undefined | parameter(), aws_config()) ->  [{topics, [[{arn, string()}]]} | {next_token, string()}]..